buckets/local/diff.go

```diff
@@ -4,14 +4,17 @@ import (
 	"context"
 	"os"
 	"path/filepath"
+	"runtime"
 	"strings"
 
 	"github.com/ipfs/go-merkledag/dagutils"
-	"github.com/logrusorgru/aurora"
+	aurora2 "github.com/logrusorgru/aurora"
 	"github.com/textileio/textile/v2/buckets"
 	"github.com/textileio/textile/v2/cmd"
 )
 
+var aurora = aurora2.NewAurora(runtime.GOOS != "windows")
```

Comment:
The main trick is basically overriding the `aurora` definition with a custom instance with coloring disabled if appropriate, instead of all places using the static methods of the package directly.